tests/Loader/LoaderIntegrationTest.php

```diff
@@ -2575,5 +2575,29 @@ public function provideFixturesToGenerate()
                 ],
             ],
         ];
+
+        yield 'method call reference value' => [
```

Comment:
Unit test with argument resolution was present.
I've added a test here with a method taking an argument that is a reference to another object.